src/components/Footer/Footer.js

```diff
@@ -1,9 +1,44 @@
 import React from 'react';
+import { INFO } from './footerData/Info';
+import { SITEMAP } from './footerData/siteMap';
```

Comment:
```suggestion
import { INFO, SITE_MAP } from './FooterData';
```
상수 데이터가 하나의 파일로 관리가 된다면 이런식으로 변경이 되겠네요 !!

Reply:
헉 도현님 이런 방법도 제시해 주셔서 너무 감사드립니다!